app/src/staging/kotlin/uk/gov/onelogin/developer/DeveloperTools.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.onelogin.developer
+
+import uk.gov.android.onelogin.BuildConfig
+
+@Suppress("KotlinConstantConditions")
+object DeveloperTools {
+    fun isDeveloperPanelEnabled() = when (BuildConfig.FLAVOR) {
+        "build" -> true
+        "staging" -> true
+        "production" -> false
+        else -> false
+    }
```

Comment:
```suggestion
    fun isDeveloperPanelEnabled() = true
```

As you are moving away from runtime check to compile check, you might as well just return true here
<img width="1437" alt="Screenshot 2024-06-27 at 13 28 18" src="https://github.com/govuk-one-login/mobile-android-one-login-app/assets/11258728/064be343-dc95-48a3-b507-1c659d9f0224">

Reply:
yep, fair point

Reply:
Same for `build` one